admin_guide/managing_networking.adoc

```diff
@@ -275,6 +275,17 @@ above example, suppose `www.foo.com` resolved to `10.11.12.13` and has a DNS TTL
 of one minute, but was later changed to `20.21.22.23`. {product-title} will then
 take up to one minute to adapt to these changes.
 
+[NOTE]
```

Comment:
@JacobTanenbaum Because the note box you're adding is to do with the step in the procedure above, it needs a + sign to link it to the paragraphs. (See line 267 for an example)